tests/Cms/Blueprints/FileBlueprintTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+use Kirby\Cms\Blueprint;
+use Kirby\Cms\File;
+use Kirby\Cms\Page;
+use Kirby\TestCase;
+
+/**
+ * @coversDefaultClass \Kirby\Cms\FileBlueprint
+ */
+class FileBlueprintTest extends TestCase
+{
+	public static function acceptAttributeProvider()
+	{
+		return [
+			[
+				'wildcard', // case name
+				'image/*', // accept option in blueprint
+				['.jpg', '.jpeg', '.gif', '.png'], // expected extensions
+				['.js', '.pdf', '.docx', '.zip'] // not expected extensions
+			],
+			[
+				'mimeAsString',
+				'image/jpeg, image/png',
+				['.jpg', '.jpeg', '.png'],
+				['.gif', '.js', '.pdf', '.docx', '.zip']
+			],
+			[
+				'mimeAsProperty',
+				['mime' => 'image/jpeg, image/png'],
+				['.jpg', '.jpeg', '.png'],
+				['.gif', '.js', '.pdf', '.docx', '.zip']
+			],
+			[
+				'extensions',
+				['extension' => 'jpg, png'],
+				['.jpg', '.png'],
+				['.gif', '.jpeg', '.js', '.pdf', '.docx', '.zip']
+			],
+			[
+				'extensionsAndMime',
+				['extension' => 'foo, bar', 'mime' => 'image/jpeg, image/png'],
+				['.jpg', '.jpeg', '.png'],
+				['.gif', '.js', '.pdf', '.docx', '.zip', '.foo', '.bar']
+			],
+			[
+				'type',
+				['type' => 'image'],
+				['.jpg', '.jpeg', '.gif', '.png'],
+				['.js', '.pdf', '.docx', '.zip']
+			],
+			[
+				'typeAndMime',
+				['type' => 'document', 'mime' => 'image/jpeg, image/png'],
+				['.jpg', '.jpeg', '.png'],
+				['.gif', '.js', '.pdf', '.docx', '.zip']
+			],
+			[
+				'intersect',
+				['type' => 'image', 'extension' => 'jpg, png, foo, bar'],
+				['.jpg', '.png'],
+				['.gif', '.js', '.pdf', '.docx', '.zip', '.foo', '.bar']
+			],
+		];
+	}
+
+	/**
+	 * @covers ::acceptAttribute
+	 * @dataProvider acceptAttributeProvider
+	 */
+	public function testAcceptAttribute($name, $accept, $expected, $notExpected)
+	{
+		Blueprint::$loaded['files/' . $name] = [
+			'accept' => $accept
+		];
+
+		$file = new File([
+			'filename' => 'tmp',
+			'parent'   => $this->createMock(Page::class),
+			'template' => $name
+		]);
+
+		$acceptAttribute = $file->blueprint()->acceptAttribute();
+
+		foreach ($expected as $extension) {
+			$this->assertStringContainsString($extension, $acceptAttribute, "Case $name: $extension should be accepted");
+		}
+
+		foreach ($notExpected as $extension) {
+			$this->assertStringNotContainsString($extension, $acceptAttribute, "Case $name: $extension should not be accepted");
+		}
+
+		unset(Blueprint::$loaded['files/' . $name]);
```

Comment:
This should happen in a `tearDown()` method so it can work even if the test fails (then this line would never be executed). Putting it in `tearDown()` also allows us to use the same file blueprint name for all cases as we can be sure that the global state will always be cleaned up in between.